packages/dd-trace/src/log/channels.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const dc = require('diagnostics_channel')
+
+const Level = {
+  Debug: 'debug',
+  Info: 'info',
+  Warn: 'warn',
+  Error: 'error'
+}
+
+const defaultLevel = Level.Debug
+
+class LogChannel extends dc.Channel {
+  constructor (name, logLevel) {
+    super(`datadog:log:${name}`)
+    this.logLevel = logLevel
+  }
+}
```

Comment:
Just wanted to point out for awareness that using the `dc.Channel` class constructor will not track the channel in the channel registry so you will be unable to look it up with another `channel(name)` call. This is fine if you pass the channels around like you do here though.

Reply:
Good to know.
 
And there is another detail: `channel.subscribe(onMessage)` is deprecated since: v18.7.0, v16.17.0

Reply:
It _is_ but I have a plan to undo that, and the reason it _was_ deprecated is not relevant if you keep a reference to the channel object.